src/charm.py

```diff
@@ -954,6 +976,70 @@ def _core_reconciliation(self):
         if self._current_wp_config() is not None:
             self._start_server()
 
+    def _wp_theme_list(self):
+        """List all installed WordPress themes"""
+        process = self._run_wp_cli(["wp", "theme", "list", "--format=json"], timeout=600)
+        if process.return_code != 0:
+            raise RuntimeError("wp theme list command failed: " + process.stderr)
+        try:
+            return json.loads(process.stdout)
+        except json.decoder.JSONDecodeError:
+            raise RuntimeError(
+                "wp theme list command failed, stdout is not json: " + repr(process.stdout)
```

Comment:
Should we use an f-string instead?

Reply:
Yes, but I heard about the f-string vulnerability recently so I don't know what to use.

Reply:
f-strings are only an issue in the case of log messages? f-strings should be fine here. Unless you are thinking of another vulnerability?

Reply:
Fixed, thanks!